### DISCUSSION TITLE
Compare performance with major browsers

### DISCUSSION THREAD

**Discussion by Nisthar** [zen-browser/desktop#644] [General]
It would be nice to see performance comparison with major browsers like ms edge.

**Comment by nocanoa** [comment]
In 2 days I will Be back and you will be able to see the scores in docs.
https://docs.zen-browser.app/benchmarks

**Reply by Nisthar** [reply]
I can't see the performance compared against other browser like edge or chrome.

**Reply by nocanoa** [reply]
yeah havent done edge or chrome, maybe tomorrow if i have time

**Comment by Nebucatnetzer** [comment]
I would like to see this as well.
If I see it correctly there is only one comparison against Firefox and there Zen was slower.
